editor/plugins/sprite_frames_editor_plugin.h

```diff
@@ -115,7 +115,8 @@ class SpriteFramesEditor : public HSplitContainer {
 	SpinBox *frame_duration = nullptr;
 	ItemList *frame_list = nullptr;
 	bool loading_scene;
-	int sel;
+	Vector<int> selections;
+	Vector<Ref<EditorSpriteFramesFrame>> frame_clipboard;
```

Comment:
I agree that I should use the resource_clipboard instead of my frame_clipboard. We can only set resource in a resource_clipboard, I'll have to go with the EditorSpriteFramesArray suggestion.

In the code I see only 4 place where we set_resource_clipboard. In the inspector dock to copy the current resource. could be anything, in the editor resource picker(can also be anything), and in the animation_library_editor (it saves an anim resource).

Reply:
Rename the EditorSpriteFramesFrame to EditorSpriteFrame (I feel it makes more sense), and created a EditorSpriteFrameArray, that inherits resources to put it in the EditorSettings clipboard.

Reply:
I've also put back the check if we have a Texture in the clipboard, and create a frame from it.